src/core_init_atmosphere/mpas_parse_geoindex.F

```diff
@@ -0,0 +1,122 @@
+module mpas_parse_geoindex
+
+   use mpas_log, only : mpas_log_write
+   use mpas_derived_types, only : MPAS_LOG_ERR, MPAS_LOG_WARN
+   use mpas_pool_routines
+
+   implicit none
+
+   private
+
+   public :: mpas_parse_index
+
+   contains
+
+   !***********************************************************************
+   !
+   !  subroutine mpas_parse_geoindex
+   !
+   !> \brief   Parse a geogrid's index file and put the results into geo_pool
+   !> \author  Miles A. Curry 
+   !> \date    03/25/2019
+   !> \details 
+   !>  Parse an index file and report type errors when they occured. Invalid types
+   !>  will cause the model to abort. Unrecognized options will be skipped. Lines 
+   !>  that start with a '#' will be treated as comments and will be ignored!
+   !>
+   !
+   !-----------------------------------------------------------------------
+   subroutine mpas_parse_index(path, geo_pool)
```

Comment:
Update to include land use index file variables including:
`category_min`
`category_max`
`iswater`
`isice`
`isurban`
`minlu`

At least to get rid of warnings.

Reply:
Also include `missing_values`